### PR TITLE
Bump cairo lang dependency

### LINKED ISSUE #350: Bump cairo-lang dependency to 0.11

### PR DESCRIPTION
**Pull request by ericnordelo**
Fixes #350

### CODE REVIEW

**Review by andrew-fleming — DISMISSED** [review]
Looks good to me!

File: setup.cfg [hunk 1]
```diff
@@ -28,8 +28,8 @@ install_requires =
     click>=8.0,<9.0
     importlib-metadata>=4.0
     python-dotenv>=0.19.2
-    starknet-devnet>=0.4.3
-    cairo-lang>=0.10.3
+    cairo-lang>=0.11.0.2
+    starknet-devnet>=0.5.0a2
```
Comment (amayvs) [inline setup.cfg:32]:
```suggestion
    starknet-devnet>=0.5.0
```

File: tox.ini [hunk 2]
```diff
@@ -16,10 +16,10 @@ extras =
 deps =
     anyio>=3.6.1
     asyncclick>=8.1.3.4
-    cairo-lang==0.10.3
-    starknet-devnet==0.4.3
+    cairo-lang==0.11.0.2
+    starknet-devnet==0.5.0a2
```
Comment (amayvs) [inline tox.ini:20]:
```suggestion
    starknet-devnet==0.5.0
```

**Review by amayvs — CHANGES_REQUESTED** [review]
@ericnordelo `starknet-devnet` released [v0.5.0](https://github.com/Shard-Labs/starknet-devnet/releases/tag/v0.5.0) 3 days ago.

Might be a worthwhile change before merge?

**Comment by amayvs** [comment]
@andrew-fleming @ericnordelo Anything I can do to assist in getting this merged and released?

**Review by andrew-fleming — APPROVED** [review]

**Comment by andrew-fleming** [comment]
@amayvs sorry for the delay, the OZ team has an event going on all week. I'll release this as soon as I can

**Pull request merged**